src/averageAges.js

```diff
@@ -55,7 +74,43 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const ageDifferences = [];
+
```

Comment:
Instead of using a loop, consider refactoring your code to use more functional programming techniques like `.filter()`, `.map()` and `.reduce()`. 🚀